jupyterhub_announcement/handlers.py

```diff
@@ -47,7 +47,7 @@ def get(self):
                 logout_url=logout_url,
                 base_url=prefix,
                 no_spawner_check=True,
-                parsed_scopes=user.get("hub_scopes") or [],
+                parsed_scopes=user.get("scopes") or [],
```

Comment:
@mahendrapaipuri Can you explain this change?

Reply:
There is no `hub_scopes` key in user model. It is `scopes`. See [here](https://github.com/jupyterhub/jupyterhub/blob/b35d77f4752e18ddd19266ceb0b647ace9004619/jupyterhub/services/auth.py#L1443). `hub_scopes` on request handler is the scopes that the user must have to access the service. In any case, we are not using `parsed_scopes` in the template for the moment so that is why the issue never manifested. In the future if we decide to use custom scopes for the service, we will need it to control the template rendering properly. Does it make sense?

Reply:
Yes!  Thank you for catching that!